### PR TITLE
clarify docs for report_participants

### PR DESCRIPTION
**Pull request by IndrajeetPatil**
The function doesn't fail if this is the case, so the docs might as well clarify what will happen in this context.

### CODE REVIEW

File: R/report_participants.R [hunk 1]
```diff
@@ -1,18 +1,22 @@
...
-#' A helper function to help you format the participants data (age, sex, ...) in the participants section.
+#' A helper function to help you format the participants data (age, sex, ...) in
+#' the participants section.
 #'
 #' @param data A data frame.
-#' @param age The name of the column containing the age.
-#' @param sex The name of the column containing the sex. Note that classes
-#'   should be one of `c("Male", "M", "Female", "F")`.
+#' @param age The name of the column containing the age of the participant.
+#' @param sex The name of the column containing the sex of the participant. The
```
Comment (DominiqueMakowski) [inline R/report_participants.R:8]: Though I suppose we could fix this behaviour?

Comment (IndrajeetPatil) [reply R/report_participants.R:8]: What approach were you thinking?

I was thinking we can check unique number of levels for `Sex` variable and then:
- if <= 2, report % females
- else, report % females and % males

What do you think?

**Comment by IndrajeetPatil** [comment]
Also, I am thinking when there are `NA`'s present in Sex column, which is often the case when surveys make entering this data optional, we can add the clause, 

> "6 participants (Mean age = 28.0, SD = 21.1, range: [8, 54]; 50.0% females of those who reported this information)"

Or something along those lines, which makes it clear that not everyone did and that there are `NA`s present.

**Comment by IndrajeetPatil** [comment]
I will merge these doc edits for now and then open a new PR to make the changes discussed in the PR.

**Pull request merged**